egs/aishell/s10/chain/options.py

```diff
@@ -25,6 +25,13 @@ def _set_inference_args(parser):
                         type=int,
                         default=-1)
 
+    parser.add_argument(
+        '--save-as-compressed',
```

Comment:
What I was talking about was mostly about changing ths command line arg to true/false and the type of the variablr to bool.

Reply:
no problem, I will change it.

Reply:
Done.